java/src/main/java/cucumber/runtime/java/AbstractJavaSnippet.java

```diff
@@ -24,7 +24,9 @@ public String arguments(List<Class<?>> argumentTypes) {
     public String tableHint() {
         return "    // For automatic transformation, change DataTable to one of\n" +
                 "    // List<YourType>, List<List<E>>, List<Map<K,V>> or Map<K,V>.\n" +
-                "    // E,K,V must be a scalar (String, Integer, Date, enum etc)\n";
+                "    // E,K,V must be a scalar (String, Integer, Date, enum etc)\n" +
+                "    // Field names for YourType must match the column names in \n" +
```

Comment:
I found some code that suggests the field name don't need to be an exact match.

`My Field`, `MyField`, `myField` will all match `myField`.

Reply:
The PascalCaseStringConverter btw.

Reply:
And also CamelCaseStringConverter.

So `my Field` will probably also work.

Reply:
Thanks, I'll have a look and rewrite a little to make this clear.